prow/config/jobs/envoy.yaml

```diff
@@ -33,6 +33,23 @@ jobs:
     value: "/home/prow/go/src/istio.io/envoy"
   command: [./ci/do_ci.sh, bazel.release]
 
+periodics:
+- interval: 168h
```

Comment:
nit: do we support something like `1w` or `7d`?

Reply:
sorry, it's golang parseduration impl, in which `Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`